client/signup/steps/design-type/index.jsx

```diff
@@ -121,6 +122,11 @@ export class DesignTypeStep extends Component {
 		this.props.recordTracksEvent( 'calypso_triforce_select_design', { category: designType } );
 
 		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], { designType } );
```

Comment:
couldn't the default themeSlugWithRepo just always be included here as a dependency?